spec/recipe_spec.rb

```diff
@@ -1,9 +1,11 @@
+# frozen_string_literal: true
+
```

Comment:
lol I chose a random file as the test for the pre-push hook and I found one with a bunch of rubocop errors :-)